src/components/inputs/money-input/money-input.spec.js

```diff
@@ -633,9 +655,7 @@ describe('MoneyInput', () => {
         language,
         options
       ) {
-        return `(${this}).toLocaleString(${language}, ${JSON.stringify(
-          options
-        )})`;
+        return new Intl.NumberFormat(language, options).format(this);
```

Comment:
Why not using react-intl to format the number?

Reply:
I don't know if they provide a similar function, that we can use here

Reply:
I would expect that to work, because react-intl builds on top of intl as far as I know. Can you look into that?

Reply:
https://github.com/yahoo/react-intl/wiki#the-react-intl-module
none of the named exports help in mocking the `toLocaleString` here
as they are targeted more to rendering the localized values into react component, not used as util functions as we need in this tests